code/modules/mob/living/carbon/carbon_say.dm

```diff
@@ -23,3 +23,13 @@
 		return T.could_speak_language(language)
 	else
 		return initial(language.flags) & TONGUELESS_SPEECH
+
+/mob/living/carbon/radio(message, list/message_mods = list(), list/spans, language)
+	if((message_mods[MODE_HEADSET] || message_mods[RADIO_EXTENSION]))
+		if(handcuffed)// If we're handcuffed, we can't press the button
+			to_chat(src, "<span class='warning'>You can't use the radio while handcuffed!</span>")
+			return ITALICS | REDUCE_RANGE
+		if(pulledby && pulledby.grab_state)
```

Comment:
`pulledby?.grab_state` acts the same as `pulledby && pulledby_grab_state`, and it's a bit neater:
```suggestion
		if(pulledby?.grab_state)
```